.github/workflows/generate.yml

```diff
@@ -1,18 +1,28 @@
-name: Generate configs and trigger network restart
+name: Generate config with delay
 
 on:
   workflow_dispatch:
     inputs:
       genesisTime:
-        description: Genesis timestamp
-        required: true
+        description: 'Genesis timestamp'
+        required: false
         default: '0'
-      network:
+      delay:
+        description: 'Delay time in minutes'
+        required: false
+        default: '0'
+      timestampMethod:
         type: choice
-        description: Network
+        options: 
+        - timestamp
+        - delay
+        description: 'Timestamp by hand (timestamp) or with certain delay (delay)'
+        required: true
+        default: 'timestamp'
```

Comment:
hmm, I think it's ok, but by default this will provide the first element as a default one:
```
       options: 
        - timestamp
        - delay
```

Reply:
default is timestamp (first option)

Reply:
ok, you can solve this conversation